client/my-sites/stats/stats-purchase/stats-purchase-wizard.jsx

```diff
@@ -0,0 +1,224 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import { Button, CheckboxControl, Card, Panel, PanelRow, PanelBody } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import React, { useState } from 'react';
+import PersonalPurchase from './stats-purchase-personal';
+import StatsPurchaseSVG from './stats-purchase-svg';
+import './styles.scss';
+
+const COMPONENT_NAME = 'stats-purchase-wizard';
+const SCREEN_TYPE_SELECTION = 0;
+const SCREEN_PERSONAL_CHECKLIST = 1;
+const SCREEN_PURCHASE = 2;
+const TYPE_PERSONAL = 'Personal';
+const TYPE_COMMERCIAL = 'Commercial';
+const FLAT_COMMERTIAL_PRICE = 10;
+
+// TODO: import
+const CommercialPurchase = () => {
+	return (
+		<div>
+			<p>Notice...</p>
+
+			<p>${ `${ FLAT_COMMERTIAL_PRICE }` }/month</p>
+
+			<div className="benefits">
+				<ul className="included">
```

Comment:
`.benefits` and `.included` may run into styling conflicts in the future.

Reply:
Yes, I'll refactor this in another PR and make all classes use the base class name const.